### PR TITLE
Layout redesign

### PR DESCRIPTION
**Pull request by Lamperoyge**
## :pushpin: References

- **Wonder issue:**
- **Related pull-requests:**

## :movie_camera: Screenshot or Video

https://www.loom.com/share/b9a21146b8134dff8d8db462debf4ef5

## :cake: Checklist:

- [x] I self-reviewed my changes
- [x] My changes follow the style guide: https://creative-earth-33e.notion.site/TT-Wonder-Style-guide-4702a45133374148953bfcaf584120b7
- [x] I tested my changes in Chrome

### CODE REVIEW

File: wondrous-app/components/Common/SidebarEntity/index.tsx [hunk 1]
```diff
@@ -1,23 +1,91 @@
...
 import RolesSidebar from 'components/Common/SidebarEntityRoles';
-import CollabsSidebar from 'components/Common/SidebarEntityCollabs';
 import { ChildrenWrapper, SidebarContent, SidebarWrapper, Wrapper } from 'components/Common/SidebarStyles';
+import { RequestApproveButton } from 'components/organization/members/styles';
+import { UnstyledLink } from 'components/WorkspacePicker/styles';
+import useMediaQuery from 'hooks/useMediaQuery';
+import { useOrgBoard, usePodBoard } from 'utils/hooks';
+import Button from 'components/Button';
+import palette from 'theme/palette';
+import { OrgInviteLinkModal } from '../InviteLinkModal/OrgInviteLink';
```
Comment (terryli0095) [inline wondrous-app/components/Common/SidebarEntity/index.tsx:18]: absolute imports

File: wondrous-app/pages/organization/[username]/members.tsx [hunk 2]
```diff
@@ -23,11 +24,20 @@ const useGetOrgFromUsername = (username) => {
...
     fetchPolicy: 'cache-and-network',
   });
   const org = useGetOrgFromUsername(username);
+  useEffect(() => {
+    if (org) {
+      setPageData({ orgData: org });
+    }
+  }, [org]);
+
+  useEffect(() => () => setPageData({}), [])
```
Comment (terryli0095) [inline wondrous-app/pages/organization/[username]/members.tsx:39]: is the point here to reset it every time?

Comment (Lamperoyge) [reply wondrous-app/pages/organization/[username]/members.tsx:39]: yes, on page change unmount we need to update the state for the dropdown to update accordingly

Comment (terryli0095) [reply wondrous-app/pages/organization/[username]/members.tsx:39]: ah

**Pull request merged**